### PR TITLE
[feature] use process fork and retrieve stack data through process

### PR DESCRIPTION
**Pull request by svnlto**

### CODE REVIEW

File: tasks/hoodie.js [hunk 1]
```diff
@@ -6,67 +6,32 @@
  * Licensed under the MIT license.
  */
 
-'use strict';
+
+var fs = require('fs');
+var util = require('util');
```
Comment (ro-ka) [inline tasks/hoodie.js]: `util` is not used.

**Comment by ro-ka** [comment]
Didn't get a notification for this PR from github. :(

File: tasks/hoodie.js (hunk 1 above)
Comment (svnlto) [inline tasks/hoodie.js]: good catch. will amend.

**Comment by gr2m** [comment]
what's the status on this one? `grunt hoodie` is broken with latest changes on hoodie-server

**Comment by ro-ka** [comment]
@svnlto: Will you amend your change? Will merge it then.

**Comment by gr2m** [comment]
I installed @svnlto's fork, but in this code

``` js
    hoodie: {
      start: {
        options: {
          callback: function(stack) {
            console.log(stack)
            grunt.config.set('connect.proxies.0.port', stack.www.port);
          }
        }
      }
    },
```

stack is

``` js
{ app: { started: true } }
```

and therefore throws error: `Fatal error: Cannot read property 'port' of undefined`

**Comment by ro-ka** [comment]
@gr2m: How does hoodie tell it's port then in the new version?

**Comment by gr2m** [comment]
I've no clue, that's beyond me :) I'm just the frontend code d00d. I hope @svnlto can help, but he's pretty busy today

**Comment by svnlto** [comment]
you'd need a more recent version of `hoodie-server` that will expose more than just `started: true`

**Comment by gr2m** [comment]
ay, that was it, sorry I forgot I use a fork of `hoodie-server` in my app.

But in the above code, `stack` is 

```
{ app: { started: true },
  pid: 17895,
  stack: 
   { couch: { port: 6042, host: '127.0.0.1' },
     www: { port: 6070, host: '127.0.0.1' },
     admin: { port: 6071, host: '127.0.0.1' } } }
```

So the grunt config example should read like

``` js
    hoodie: {
      start: {
        options: {
          callback: function(config) {
            grunt.config.set('connect.proxies.0.port', config.stack.www.port);
          }
        }
      }
    },
```

Cheers!

**Comment by svnlto** [comment]
:thumbsup:

**Comment by ro-ka** [comment]
@svnlto: Will you update the PR?

**Comment by svnlto** [comment]
should be GTG now.

**Comment by ro-ka** [comment]
@svnlto: The `Gruntfile.js` needs to be updated, too. And the not used `util` (see above) can be removed.

**Comment by svnlto** [comment]
ftfy

**Comment by ro-ka** [comment]
Published as version `0.2.0`.

**Pull request merged**